services/121-service/src/main.ts

```diff
@@ -71,6 +73,39 @@ function generateModuleDependencyGraph(app: INestApplication): void {
   });
 }
 
+function generateSwaggerSummaryJson(app: INestApplication<any>): void {
+  const options = new DocumentBuilder()
+    .setTitle(APP_TITLE)
+    .setVersion(APP_VERSION)
+    .build();
+  const openApiDocument = SwaggerModule.createDocument(app, options);
+
+  const minimalDocument: {
```

Comment:
Could we also add the parameters to this object?

Reply:
Endpoint params*

Reply:
Ah yes I missed as I only was using the 200 code. Good one